src/protect-function-app/README.md

```diff
@@ -0,0 +1,124 @@
+| **Client secret (recommended)** | _None_                                                  | Suggested value for this sample. <br/> This sample doesn't require this feature. |
+| **Issuer URL**                  | `https://login.microsoftonline.com/<tenant-id>/v2.0`    | Required value for this sample. <br/> Update to include 'Tenant ID' of your Azure AD instance - this value is a GUID                     |
+| **Allowed token audiences**     | `api://<client-id>`                                     | Required value for this sample. <br/> 'Application ID URI' of app registration in Azure portal - this value typically starts with api:// |
+| **Restrict access**             | **Require authentication**                              | Required value for this sample.                                                  |
+| **Unauthenticated requests**    | **HTTP 401 Unauthorized: recommended for APIs**         | Suggested value for this sample.                                                 |
+| **Token store**                 | _Unselected_                                            | Suggested value for this sample.                                                 |
+
+> :information_source: **Bold text** in the table matches (or is similar to) a UI element in the Azure portal, while `code formatting` indicates a value you enter into a text box in the Azure portal.
+
+### 3. Deploy the Function app
+
+```console
+func azure functionapp publish <your-Function-App-name>
+```
+
+## Access the API
+
+Using Postman, curl, or a similar application, issue an HTTP GET request to https://\<your-function-app-name>.azurewebsites.net/api/greeting with an `Authorization` header of `Bearer {VALID-ACCESS-TOKEN}`.
+
+For example, if you use curl and everything worked, you should receive a response from the Azure Function similar to this.
+
+```console
+$ curl https://<your-function>.azurewebsites.net/api/greeting -H "Authorization: Bearer <VALID-ACCESS-TOKEN>"
+Hello, world. You were able to access this because you provided a valid access token with the Greeting.Read scope as a claim.
+```
+
+## About the code
+
+This Azure Function is an anonymous HTTP trigger written in csharp and uses the built-in [Authentication and authorization in Azure Functions](https://docs.microsoft.com/azure/app-service/overview-authentication-authorization) feature to offload fundamental JWT access token validation. Requests that make it through the built-in authentication feature of Azure Functions are then routed to the csharp code, which applies additional access token validation checking for a specific scope.
+
+- A missing or invalid (expired, wrong audience, etc) token will result in a `401` response. (Handled by Azure Functions authentication)
+- An otherwise valid token without the proper scope will result in a 403 response.
```

Comment:
```suggestion
- An otherwise valid token without the proper scope will result in a `403` response.
```

Just to make the styling consistent with the `401` response

Reply:
thanks for this review @skabou, definitely accepting this contrib.